smdebug/core/utils.py

```diff
@@ -64,24 +64,29 @@ class FRAMEWORK(Enum):
 except (ImportError, ModuleNotFoundError):
     _torch_dist_imported = None
 
-
+logger = get_logger()
 try:
     import horovod.torch as hvd
 
     # This redundant import is necessary because horovod does not raise an ImportError if the library is not present
     import torch  # noqa
+    #make sure the library is correctly imported
+    hvd.init()
```

Comment:
Do we know if this could interfer with other distribution strategies? I'm worried running init on horovod might cause a problem if the user is running SMDDP or DDP.